contents/docs/product-analytics/sharing.mdx

```diff
@@ -8,20 +8,22 @@ export const sharePubliclyDark = "https://res.cloudinary.com/dmukukwp6/image/upl
 export const embedInsightLight = "https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/docs/product-analytics/embed-insight-light-mode.png"
 export const embedInsightDark = "https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/docs/product-analytics/embed-insight-dark-mode.png"
 
-You can share a public link to a specific insight or embed an insight in a web page or application (where the insight will be displayed in an iframe).
+For any dashboard or insight, you can share a public link and/or embed it using an iframe. This is useful if you want to share your data with customers or partners, but don't want to share your entire account.
 
-## Share an insight's public link
+> **Note:** When you share an insight or dashboard, anyone with the link can view it. To better control this data, consider setting up [customer-facing analytics](/tutorials/customer-facing-analytics).
 
-1. Choose _Share or embed_ from the `...` (more options) menu located at the top right corner of the page.
+## Sharing an insight
 
-<ProductScreenshot
+To share an insight, click the `...` (more options) menu next to its name and choose "Share or embed."
```

Comment:
Nit: I prefer to bold button titles instead of using quotes. It makes it clearer
```suggestion
To share an insight, click the `...` (more options) menu next to its name and choose **Share or embed**.
```